### DISCUSSION TITLE
Add support of the cursor pagination

### DISCUSSION THREAD

**Discussion by vrusua** [tailflow/laravel-orion#299] [Ideas]
@alexzarbn 

Hi Aleksei,

Recently, the cursor pagination has been added to Laravel core and Eloquent API Resources:
https://laravel.com/docs/8.x/pagination#cursor-pagination
https://github.com/laravel/framework/pull/37216
https://github.com/laravel/framework/pull/37315

It suits better for infinite scrolling and has a better performance in comparison to the offset based one
https://shopify.engineering/pagination-relative-cursors

Just wondering, can we use the cursor pagination in Orion, or?

Thanks,
Ruslan

**Comment by alexzarbn** [comment]
Hi @vrusua,

It is currently not supported in Orion, but might be added in the future releases.

In the meantime, please take a look at how you can [customize queries](https://tailflow.github.io/laravel-orion-docs/v2.x/guide/models.html#customizing-queries) - might be possible to get cursor pagination working by overriding a few query builder methods.
